tinyagentos/routes/channel_hub.py

```
@@ -1,10 +1,58 @@
+            _connect_locks.pop(connector_key, None)
+
+
+async def _stop_prior_connector(connectors: dict, connector_key: str) -> None:
+    """Stop and drop any prior connector under this key. Reconnecting (e.g.
+    after a token rotation) must not silently orphan the previous connector's
+    background task/client."""
+    old = connectors.pop(connector_key, None)
+    if old is not None and hasattr(old, "stop"):
+        try:
+            await old.stop()
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)